examples/source_examples/oauth2_with_token_refresh/readme.md

```diff
@@ -0,0 +1,40 @@
+# OAuth2 Refresh Token Hubspot Connector Example
+
+> NOTE: We have Fivetran connector for HubSpot that users can integrate directly on the dashboard [here](https://fivetran.com/docs/connectors/applications/hubspot#hubspot). You can use this example as a reference for the cases where you want to connect to a custom OAuth2 source that requires access token refresh.
+
+## Prerequisites
+
+1. If you don't have a HubSpot account, follow HubSpot's [instructions](https://developers.hubspot.com/docs/guides/apps/public-apps/overview) to create one.
+
+2. Create a developer account and a HubSpot app with scopes and redirect URL. See HubSpot's [OAuth documentations](https://developers.hubspot.com/docs/reference/api/app-management/oauth) for details.
+
+3. Use this above credentials in Postman inside Authroization->OAuth2.0 as shown in the below ss:
```

Comment:
@fivetran-pranavtotala what does this mean?
"Use this above credentials" - I believe we need to list the credentials for when image may be broken